test/id.test.js

```diff
@@ -18,8 +18,7 @@ describe('using _id as id', function() {
       owner: {type: String},
       _id: {type: String, id: true},
     }, {forceId: true});
-
-    db.automigrate('Task', done);
```

Comment:
Is removing the `db.automigrate` intentional? (Seems unrelated to the issue -- just want to confirm it wasn't accidental).

Reply:
@virkt25 1.x has a bug with `automigrate` function, which is fixed in 2.x, if you look at all the 1.x tests, they don't call `automigrate`, calling it results in a document conflict error.

Reply:
find the related PR:
https://github.com/strongloop/loopback-connector-cloudant/commit/e348c2576d7fead66ce7370871fe5cfecef56539
and 
https://github.com/strongloop/loopback-connector-cloudant/pull/109
if you are interested.

Reply:
Ooh ok. I just wanted to make sure it wasn't accidental and I didn't realize this was for 1.x branch